File: PYTHON/Program/Ssd.py
from Komponen import Komponen

class Ssd(Komponen):
  def __init__(self, kapasitasGB=0, tipeDrive="", merk="", nama=""):
    super().__init__(merk, nama)
    self.kapasitasGB = kapasitasGB
    self.tipeDrive = tipeDrive

  def setKapasitasGB(self, kapasitasGB):
    self.kapasitasGB = kapasitasGB

  def setTipeDrive(self, tipeDrive):
    self.tipeDrive = tipeDrive

  def getKapasitasGB(self):
    return self.kapasitasGB

  def getTipeDrive(self):
    return self.tipeDrive

File: JAVA/Program/Ssd.java
public class Ssd extends Komponen {
  private int kapasitasGB;
  private String tipeDrive;

  public Ssd() {
  }

  public Ssd(int kapasitasGB, String tipeDrive, String merk, String nama) {
    super(merk, nama);
    this.kapasitasGB = kapasitasGB;
    this.tipeDrive = tipeDrive;
  }

  public void setKapasitasGB(int kapasitasGB) {
    this.kapasitasGB = kapasitasGB;
  }

  public void setTipeDrive(String tipeDrive) {
    this.tipeDrive = tipeDrive;
  }

  public int getKapasitasGB() {
    return this.kapasitasGB;
  }

  public String getTipeDrive() {
    return this.tipeDrive;
  }
}
